server/jetty/README.md

```diff
@@ -1,5 +1,44 @@
 # Native packaging for Deephaven Jetty server
 
+### Setting up a Python virtual environment
+
+This is an optional prerequisite, but lets the later commands to run the server work without
+specifying groovy as the console language. If you skip this step, be sure to use groovy, or
+there will be errors on startup indicating that the Python environment is not suitable.
+
+See https://github.com/deephaven/deephaven-core/issues/1657 for more discussion
+
+1. On MacOS there is an extra patch to apply at this time, to add an extra argument to clang,
+see the first code snippet of the comment at
+https://github.com/deephaven/deephaven-core/issues/1657#issuecomment-989040798 for specifics.
+1. Make sure Python is installed with a shared library. For example, if using `pyenv install`,
+be sure to first set `PYTHON_CONFIGURE_OPTS="--enabled-shared"`.
+1. Make a new directory for a virtual environment, and set it up:
+    ```shell
+    $ mkdir dh-py && cd dh-py
+    $ python -m venv local-jetty-build
+    $ source local-jetty-build/bin/activate # this must be re-run for each new shell
+    $ cd -
+   ```
+1. Build and install wheels for deephaven-jpy and deephaven:
+    ```shell
+    $ python -m pip install --upgrade pip # First upgrade pip
+    $ pip install wheel
+    $ export DEEPHAVEN_VERSION=0.8.0 # this should match the current version of your git repo
+
+    $ cd py/jpy
+    $ JAVA_HOME=/path/to/your/java/home # Customize this to fit your computer
+   $ python setup.py bdist_wheel
```

Comment:
Unfortunately, this is breaking for me (Fedora 35 running on python 3.10.1), so I'm not able to test locally.

```
src/main/c/jpy_jobj.c: In function ‘JType_InitSlots’:
src/main/c/jpy_jobj.c:714:24: error: lvalue required as left operand of assignment
  714 |     Py_REFCNT(typeObj) = 1;
```

Probably time to incorporate https://github.com/jpy-consortium/jpy/pull/14?

Reply:
We don't explicitly support any python version that we don't package at this time (and probably we should actually test the matrix of packages we ship before getting into testing cross OS, py vers, arch...)

Reply:
We only "package" python 3.7, right? In your example though, it looks like you are building w/ 3.9? 

It would be nice to get back into more native packaging / releasing of artifacts as wheels with each release; in which case we could be very explicit about what we expect should work.

Reply:
We only package 3.7 yes, and it happens to work with 3.9 too as far as I can tell. `pyenv` probably should be used for now to try out a different version.